composeApp/build.gradle.kts

```diff
@@ -41,6 +41,9 @@ kotlin {
             implementation(libs.decompose)
             implementation(libs.decompose.extensions.compose)
             implementation(libs.serialization.json)
+
+            implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.8.0")
```

Comment:
I guess we don't need to add `kotlinx-coroutines-core` dependency? It should be already coming from Compose transitively.

Reply:
Hello, thanks for code-review, check it now!